cmd/commands/image.go

```diff
@@ -60,8 +61,8 @@ func ImageRelocate(c *core.Client) *cobra.Command {
     ... 
 
 `,
-		Example: `  riff image relocate --manifest=/path/to/manifest --registry=hostname --user=username --images=/path/to/image/manifest --output=/path/to/output/dir
-  riff image relocate --file=/path/to/file --registry=hostname --user=username --images=/path/to/image/manifest --output=/path/to/output`,
+		Example: `  riff image relocate --manifest=/path/to/manifest.yaml --registry=hostname --user=username --images=/path/to/image-manifest.yaml --output=/path/to/output/dir
```

Comment:
I'd change those examples to relative paths, even though both are supported, because this seems like the most typical use